deliverables/homework.playground/Contents.swift

```diff
@@ -1,18 +1,131 @@
-//: Shopping List - Week 1
-
 import Foundation
 
-//: Create an empty array to store shopping list items
+//Walking through homework assignments step-by-step
+
+
+// Question #1) 
+//=============
```

Comment:
I like your styling! I do this a lot with my code which really helps as files get really long and complicated. Keep this up, it will keep you mind happy and organized :+1: